money-monkey-web/components/GoogleSignInButton.tsx

```
@@ -0,0 +1,43 @@
+  if (typeof window !== "undefined") {
+    window.handleSignInWithGoogle = (response: {
+      credential: string;
+      select_by: string;
+      state?: string;
+    }) => signIn(response.credential);
+  }
+
+  return (
+    <div>
+      {/* google sign in  */}
+      <Script src="https://accounts.google.com/gsi/client" async></Script>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)